tests/integration/ha/test_ha.py

```diff
@@ -680,11 +680,11 @@ async def test_multi_clusters_db_isolation(
     await ops_test.model.wait_for_idle(
         apps=[app],
         status="active",
-        timeout=1000,
+        timeout=3600,
```

Comment:
Silly small thing but don't we wanna have a constant for this? :-)
(Feel free to ignore :-) )